frontend/src/components/user_page/components/UserHeader.tsx

```diff
@@ -0,0 +1,27 @@
+import { FunctionComponent } from "react";
+import { User } from "../../../shared/Interfaces";
+
+interface Props {
+  userProps?: User;
+}
+
+const UserHeader: FunctionComponent<Props> = ({ userProps }) => {
+  return (
+    <div>
+      <div className="flex items-center">
+        <img
+          src={userProps ? userProps.avatarUrl : ""}
+          className="w-15 h-15 block rounded  "
```

Comment:
I don't think `w-15 h-15` is a recognized size in tailwind, maybe try `w-14 h-14` ?

Reply:
<img width="1268" alt="Screen Shot 2022-03-17 at 8 23 35 PM" src="https://user-images.githubusercontent.com/8432061/158931672-4197fe4a-787c-4bf7-bce1-5a4eadcb2b8b.png">